assets/dev-env.nginx.template.conf.ejs

```diff
@@ -0,0 +1,13 @@
+<% if ( mediaRedirectDomain ) { %>
+
+location ~* \.(png|jpg|jpeg|gif|ico)$ {
```

Comment:
I think we should try to find out what we have in the location block for static assets and try to match that.

Reply:
That is what we have in the default config - https://github.com/Automattic/vip-container-images/blob/9194f47bbf3d5832501456dad19d4e68ee8ce032/nginx/default.conf#L59

Or what do you mean?

Reply:
I tested with  `/wp-content/uploads` and it works great.